promgen/static/js/promgen.vue.js

```diff
@@ -368,7 +368,13 @@ app.component('silence-list-modal', {
     },
     computed: {
         activeSilences() {
-            return this.$root.activeSilences || [];
+            silences = this.$root.activeSilences || []
+            if (silences) {
+                for (let silence of silences) {
```

Comment:
You are not going to modify the `silence` reference (`silence = something`), therefore, please use `const`:

```
for (const silence of silences) {
```